test/e2e/tests/releases/displayDocument/DisplayedDocument.spec.ts

```diff
@@ -0,0 +1,445 @@
+  discardVersion,
+  getRandomReleaseId,
+} from '../utils/methods'
+
+// for before all to work with single worker and run only once per tests and describe
+// this is to avoid issues with multiple releases being created per test
+test.describe.configure({mode: 'serial'})
+
+test.describe('displayedDocument', () => {
+  let publishedDocument: SanityDocument
+  let publishedWithVersion: SanityDocument
+  let publishedDocumentDupe: SanityDocument
+  let versionDocumentOne: SanityDocument
+  let versionDocumentTwo: SanityDocument
+
+  let asapReleaseId: string
+  let undecidedReleaseId: string
+
+  let multipleVersionsDocId: string
+  let singleASAPVersionDocument: SanityDocument
+
+  test.beforeAll(async ({sanityClient, _testContext, browserName}) => {
+    test.skip(browserName === 'firefox', 'Skip test outside of chromium due to flakiness')
+    publishedDocument = await createDocument(sanityClient, {
+      ...speciesDocumentNamePublished,
+      _id: `${_testContext.getUniqueDocumentId()}`,
+    })
+
+    const dataset = sanityClient.config().dataset
+
+    asapReleaseId = getRandomReleaseId()
+    undecidedReleaseId = getRandomReleaseId()
+
+    // create releases
+    await createRelease({
+      sanityClient,
+      dataset,
+      releaseId: asapReleaseId,
+      metadata: partialASAPReleaseMetadata,
+    })
+
+    await createRelease({
+      sanityClient,
+      dataset,
+      releaseId: undecidedReleaseId,
+      metadata: partialUndecidedReleaseMetadata,
+    })
+
+    singleASAPVersionDocument = await createDocument(sanityClient, {
+      ...speciesDocumentNameASAP,
+      _id: `versions.${asapReleaseId}.${_testContext.getUniqueDocumentId()}`,
+    })
+
+    multipleVersionsDocId = _testContext.getUniqueDocumentId()
+
+    versionDocumentOne = await createDocument(sanityClient, {
+      ...speciesDocumentNameASAP,
+      _id: `versions.${asapReleaseId}.${multipleVersionsDocId}`,
+    })
+
+    versionDocumentTwo = await createDocument(sanityClient, {
+      ...speciesDocumentNameUndecided,
+      _id: `versions.${undecidedReleaseId}.${multipleVersionsDocId}`,
+    })
+
+    const publishedIdWithVersions = _testContext.getUniqueDocumentId()
+
+    publishedDocumentDupe = await createDocument(sanityClient, {
+      ...speciesDocumentNamePublished,
+      _id: `${publishedIdWithVersions}`,
+    })
+
+    publishedWithVersion = await createDocument(sanityClient, {
+      ...speciesDocumentNameASAP,
+      _id: `versions.${asapReleaseId}.${publishedIdWithVersions}`,
+    })
+  })
+
+  test.afterAll(async ({sanityClient, browserName}) => {
+    test.skip(browserName === 'firefox', 'Skip test outside of chromium due to flakiness')
+    const dataset = sanityClient.config().dataset
+
+    await Promise.all([
+      archiveAndDeleteRelease({sanityClient, dataset, releaseId: asapReleaseId}),
+      archiveAndDeleteRelease({sanityClient, dataset, releaseId: undecidedReleaseId}),
+    ])
+
+    //await deleteAllReleases({sanityClient, dataset})
```

Comment:
Should this be here? If so, maybe add a comment saying what it's for?

Reply:
🤦 forgot to remove this, you saw nothing 🥷 
sorry about the noise